### PR TITLE
Toast

### PR DESCRIPTION
**Pull request by Jayesh3636**
- Added Toast

https://user-images.githubusercontent.com/91378157/158459252-cb2e89e5-702f-4054-a788-f2e08635bc07.mp4

Can anyone please review it!
let me know the changes if required!
TIA~!

### CODE REVIEW

File: Components/toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+.toast-error {
+    color: #dc3545;
+    background-color: #fcc5c5;
+}
+
+.toast-inform {
+    background-color: #7FD2E6;
+    color: #29438b;
```
Comment (anjali1102) [inline Components/toast/toast.css:38]: using variable whenever color.
 can we do like this.

File: Components/toast/toast.html [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+            </div>
+        </nav>
+    </header>
+    <main class="grid-layout">
+        <!-- first page -->
+        <div class="bar"></div>
+        <div class="CUBE1"></div>
+        <div class="CUBE2"></div>
+        <div class="CUBE3"></div>
+        <div class="CUBE4"></div>
```
Comment (anjali1102) [inline Components/toast/toast.html:61]: empty divs, what are you trying to do with these

File: Components/toast/toast.css (hunk 1 above)
Comment (anjali1102) [reply Components/toast/toast.css:38]: code formatting

**Review by anjali1102 — CHANGES_REQUESTED** [review]
hey Jayesh
small suggestions 😀

File: Components/toast/toast.css [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    background-color: white;
+    color: var(--secondary-text-color);
+    border-radius: 0.6rem;
+    display: none;
+    z-index: 5;   
+} 
+
+
+.toast-success {
+    background-color: #84f1c9;
```
Comment (Kavita539) [inline Components/toast/toast.css:27]: Use global variables for colors to have consistency throughout the code.

**Review by Kavita539 — COMMENTED** [review]
Just a small change suggested. Other than that code is well structured and looks good. Loved how you have mentioned comments as it makes the review easy.

File: Components/toast/toast.css (hunk 1 above)
Comment (Jayesh3636) [reply Components/toast/toast.css:38]: Hey there~!
thanks for giving your valuable time in reviewing my project!

Color Variables:
I know we have to use color variables whenever needed but I have used all these 6 colors distinctively in toast CSS only.
and it's getting too heavy to manage colors in reset CSS or global CSS. Maybe in the future, I'll make it your way!

File: Components/toast/toast.html (hunk 2 above)
Comment (Jayesh3636) [reply Components/toast/toast.html:61]: Actually, if you see it clearly this isn't just empty Divs, these are the design elements in UI, the class CUBE # is mentioned in library.css where the template has been written! if you wanna know more check it here 

https://github.com/Jayesh3636/Dark-Bucket/blob/dev/Components/library.css

File: Components/toast/toast.css (hunk 3 above)
Comment (Jayesh3636) [reply Components/toast/toast.css:27]: Hey, Kavita thanks for putting your time into reviewing this project!
Thanks for the appreciation! and like I have told Anjali above, I have used all these 6 colors distinctively in toast CSS only. and it's getting too heavy to manage colors in reset CSS.

**Comment by Jayesh3636** [comment]
thanks for your reviews!

**Pull request merged**